hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/S3LifecycleConfiguration.java

```diff
@@ -257,13 +257,14 @@ public OmLifecycleConfiguration toOmLifecycleConfiguration(OzoneBucket ozoneBuck
       OmLifecycleConfiguration.Builder builder = new OmLifecycleConfiguration.Builder()
           .setVolume(ozoneBucket.getVolumeName())
           .setBucketLayout(ozoneBucket.getBucketLayout())
+          .setCreationTime(System.currentTimeMillis())
```

Comment:
Can I check which creation time is used? This one or the one in `OMLifecycleConfigurationSetRequest`?

Reply:
This can be removed, OMLifecycleConfigurationSetRequest will set the `creationTime` too.